extensions/wikia/WikiFactory/WikiFactory.php

```
@@ -1273,6 +1273,15 @@ static public function getWikisByID( $ids, $master = false ) {
 			return [];
 		}
 
+		if ( !is_array( $ids ) ) {
+			WikiaLogger::instance()->debug( "WikiFactory::getWikisByID supplied with a non-array argument", [
+				'exception' => new Exception(),
+				'arguments_ids' => $ids,
+			]);
+
+			return [];
+		}
+
```

Let's fail early and force `Array` as `$ids` argument type.